source/val/validate_barriers.cpp

```diff
@@ -45,12 +45,15 @@ spv_result_t BarriersPass(ValidationState_t& _, const Instruction* inst) {
                       model != SpvExecutionModelGLCompute &&
                       model != SpvExecutionModelKernel &&
                       model != SpvExecutionModelTaskNV &&
-                      model != SpvExecutionModelMeshNV) {
+                      model != SpvExecutionModelMeshNV &&
+                      model != SpvExecutionModelTaskEXT &&
+                      model != SpvExecutionModelMeshEXT) {
```

Comment:
SPV_EXT_mesh_shader requires SPIR-V 1.4. This check only happens if the binary version is less than 1.3.

Reply:
The validation for the 1.4 requirement appears to be missing. Please add it to validate_extensions.cpp.

Reply:
regarding 1.4 requirement added code to validate_extensions.cpp::ValidateExtension
and removed changes from validate_barriers.cpp